### DISCUSSION TITLE
The new GUI is worse than the old one.

### DISCUSSION THREAD

**Discussion by pawa2401** [Blinue/Magpie#663] [General]
In my opinion, the GUI introduced after v0.10 is badly thought out and harder to use than the v0.9 one.

I don't have any trouble with technology, but the new GUI managed to confuse me. In my opinion, profiles make sense when you have a lot of settings that have to be changed for multiple apps, but from my personal experience, the only 2 settings I changed regularly were Capture method and Scaling algorithm. Sometimes GDI works better than graphics capture, but it's incompatible with UWP apps.

It's a real hassle to set up profiles for each program, when you could just change the setting you want in 2 seconds.
I love the old GUI for its simplicity. All the most basic things are kept on the main UI, and all the other settings are in the settings UI. It feels way more manageable.

Another big change for me was the keybind system change. What was wrong with it? I never had ANY issues with it, apart from the fact that you had to type the keybind manually, but once I figured it out the possibilities were endless.

I love this app. It helps me run games with higher settings at lower resolutions while maintaining good looks thanks to the scaling algorithms and filling the screen completely. But I feel that this project is going in a bad direction in terms of ease of use, even for a decently capable user.

I don't have any real coding experience outside of game scripting, but If needed I could contribute to this project by trying to make some concepts of how the UI could work to become more usable, though just modernizing the old UI would be enough.

**Comment by Blinue** [comment]
Thank you for your feedback. Let me summarize it:

1. **If you don’t use profiles, changing the capture method and scaling mode is more troublesome than v0.9.**
That’s true. Now you have to go to the “Defaults” page first, which adds one more mouse click than before.

3. **Non-standard shortcuts are no longer supported.**
This issue has been discussed in #627.

I personally prefer the new UI because it has a modern look, but of course there are also problems, such as slow response, bloatedness, etc., but there is no perfect UI framework.

It is impossible to abandon the new UI, but I am happy to make improvements in terms of usability, such as allowing to change global settings through the tray icon #442

**Reply by pawa2401** [reply]
Sad to see the shortcuts go, but thanks for caring about improving this. I wish you best of luck in improving this new UI.
